### ISSUE TITLE
DataFrame.drop() takes from 1 to 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by santivill** [bmabey/pyLDAvis#256]
When I execute this:
drop_cols = ['date', 'volume', 'open', 'low', 'high']
train_df = train_df.drop(drop_cols, 1)
valid_df = valid_df.drop(drop_cols, 1)
test_df  = test_df.drop(drop_cols, 1)

I got the error:
DataFrame.drop() takes from 1 to 2 positional arguments but 3 were given

**Comment by Lehas-sudo** [comment]
I am a beginner but I had the same problem. I made it work for me, but if somebody with more knowledge can confirm please do so. the error is in the drop function used in the prepare.py file.

you need to find the prepare.py file and edit following code passage:
lib/python3.8/site-packages/pyLDAvis/_prepare.py

```
--> 243 default_term_info = default_term_info.sort_values(
    244     by='saliency', ascending=False).head(R).drop('saliency', 1)
```

you have to add the axis argument in the file, restart your Jupyter kernel and then it should work.

```
--> 243 default_term_info = default_term_info.sort_values(
    244     by='saliency', ascending=False).head(R).drop('saliency', **axis =** 1)
```

**Comment by SoloSynth1** [comment]
This is fixed in current release (3.4.1). Upgrading the package should resolve this issue.
https://github.com/bmabey/pyLDAvis/blob/997bd7cf26efefd60a5d9915ee22542fd9ca6624/pyLDAvis/_prepare.py#L244
